config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
   root 'welcome#index'
 
+  get '/activities', to: "activities#index"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.